lib/origen_testers/smartest_based_tester/base/test_methods/limits.rb

```diff
@@ -28,13 +30,17 @@ def unit=(val)
 
           def to_s
             if !lo_limit && !hi_limit
-              "\"#{test_name}\"" + ' = "":"NA":"":"NA":"":"":""'
+              if tnum == ""
```

Comment:
Oh yeah, sorry I missed the 0 at the end.

Good update btw, I've been wondering how test numbers should be handled on 93k.